transcom/mymove
# INT-B-17922 - Deprecate INTERNAL PPM APIs

## [B-17922](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A868513&RoomContext=TeamRoom%3A855664&concept=TeamRoom)

## Summary

**INTEGRATION**

Removing internal endpoints(PPM) no longer in use.

GET: /estimates/ppm:
      operationId: showPPMEstimate
      tags:
        - ppm

 GET: /estimates/ppm_sit:
      operationId: showPPMSitEstimate
      tags:
        - ppm

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

Best way to verify is to run client apps locally and go to the swagger page. The APIs will no longer be exposed.
`http://milmovelocal:3000/swagger-ui/internal.html`
`GET: /estimates/ppm`
`GET: /estimates/ppm_sit`
